HintList.py

```diff
@@ -557,17 +557,17 @@ def tokens_required_by_settings(world: World) -> int:
     'HF GS Cow Grotto':                                            ("a #spider behind webs# in a grotto holds", None, ['overworld', 'sometimes']),
     'HF Cow Grotto Cow':                                           ("the #cobwebbed cow# gifts", "a #cow behind webs# in a grotto gifts", ['overworld', 'sometimes']),
     'ZF GS Hidden Cave':                                           ("a spider high #above the icy waters# holds", None, ['overworld', 'sometimes']),
-    'Wasteland Chest':                                             (["#deep in the wasteland# is", "beneath #the sands#, flames reveal"], None, ['overworld', 'sometimes']),
+    'Wasteland Chest':                                             (["#deep in the wasteland# is", "beneath #the sands#, flames reveal"], "a chest spawned by #lighting flames in Haunted Wasteland# contains", ['overworld', 'sometimes']),
     'Wasteland GS':                                                ("a #spider in the wasteland# holds", None, ['overworld', 'sometimes']),
-    'Graveyard Royal Familys Tomb Chest':                          (["#flames in the royal tomb# reveal", "the #Composer Brothers hid#"], None, ['overworld', 'sometimes']),
-    'ZF Bottom Freestanding PoH':                                  ("#under the icy waters# lies", None, ['overworld', 'sometimes']),
-    'GC Pot Freestanding PoH':                                     ("spinning #Goron pottery# contains", None, ['overworld', 'sometimes']),
+    'Graveyard Royal Familys Tomb Chest':                          (["#flames in the royal tomb# reveal", "the #Composer Brothers hid#"], "#lighting flames in the royal tomb# rewards", ['overworld', 'sometimes']),
+    'ZF Bottom Freestanding PoH':                                  ("#under the icy waters# lies", "at the #bottom of the lake in Zora's Fountain# lies", ['overworld', 'sometimes']),
+    'GC Pot Freestanding PoH':                                     ("spinning #Goron pottery# contains", "putting an explosive in a #happy face Goron Pot# rewards", ['overworld', 'sometimes']),
```

Comment:
I thing this should be rephrased, but I'm not exactly sure how, as its a bit imprecise here. Something like "putting an explosive in the huge Goron pot until it's happy"?

Reply:
"The Goron Pot's happy/angry/neutral face spits out"?